testgrid/config.yaml

```diff
@@ -2545,12 +2561,20 @@ test_groups:
 #              periodic strigger ci-periodic-${zuul-job-name}
 - name: ci-presubmit-cloud-provider-openstack-acceptance-test-e2e-conformance
   gcs_prefix: kubernetes-openstack/pr-logs/ci-cloud-provider-openstack-acceptance-test-e2e-conformance
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 1   
```

Comment:
/cc @dims 
you might want alerts for these :-)

Reply:
+1 yes please. @AishSundar could you please throw my email id to the mix (or i can do it in a follow up PR) davanum AT gmail.com